optimizely/project_config.py

```diff
@@ -498,8 +499,12 @@ def set_forced_variation(self, experiment_key, user_id, variation_key):
       # The invalid experiment key will be logged inside this call.
       return False
 
+    if variation_key is not None and not validator.is_non_empty_string(variation_key):
```

Comment:
Move this _not validator.is_non_empty_string(variation_key):_
after _variation_key is None block_